enable_retired_username_email/signals.py

```diff
@@ -0,0 +1,72 @@
+"""
+Signal handler for setting setting modified username and email hash in user model.
+"""
+
+import time
+
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from enable_retired_username_email.compat import get_retired_email_hash, get_user_retirement_status_model
+
+UserRetirementStatus = get_user_retirement_status_model()
+
+
+@receiver(post_save, sender=UserRetirementStatus)
+def replace_username_email_hash(
+    instance: UserRetirementStatus,
+    created,
```

Comment:
Type hinting for created needs to be added

Reply:
Done